### DISCUSSION TITLE
#Request Fitur Baru

### DISCUSSION THREAD

**Discussion by fajar-dev** [fajar-dev/dirmahait-v.2#19] [Ideas]
tif-unimal20.web.id

<hr /><em>This discussion was created from the release <a href='https://github.com/fajar-dev/dirmahait-v.2/releases/tag/'>#UNIMAL</a>.</em>

**Comment by scotch01** [comment]
Request fitur chat

**Reply by fajar-dev** [reply]
noted gan
